doc/installation/install.rst

```diff
@@ -1,79 +0,0 @@
-.. highlight:: sh
-.. _installation-install:
-  
-Installing Zotonic
-==================
-
-Before beginning installation, please be sure your system meets the
-:ref:`installation-preinstall`.
-
-At the end of this page, you should have a bare-bones Zotonic running,
-without any sites in it.
-
-There are multiple ways of installing Zotonic, depending on what you want and 
-
-1. :ref:`installation-install-release`
-2. :ref:`installation-install-git`
-3. :ref:`installation-install-script`
-
-For most people, :ref:`installation-install-release` should be the best option.
-   
-
-.. _installation-install-release:
-   
-Installing from a release .zip archive
---------------------------------------
-
-Zotonic versions are released as `.zip` files from the official
-download page. This installation method gets you the officially
-supported version.
-
-1. Download the latest release as a `.zip` from
-   https://github.com/zotonic/zotonic/releases, and unzip it in a
-   folder called ``zotonic``.
-
-.. include:: _make_common.rst
-
-.. _installation-install-git:
-
-Installing from `git`
-----------------------------------------------
-
-If you want to run the latest version of Zotonic or run on one of its
-release branches, get a copy of the source tree from github.com and
-run Zotonic from there.
-
-1. Use `git <http://git-scm.com/>`_ to `clone` a copy of the Zotonic source tree::
-
-     git clone git://github.com/zotonic/zotonic.git
-     cd zotonic
-
-   You are now in the master branch, which contains the current
-   development version of Zotonic. To switch to the more stable 0.12
-   version, switch to the ``release-0.12.x`` branch::
-
-     git checkout release-0.12.x
-
-.. include:: _make_common.rst
-
-             
-.. _installation-install-script:
-
-Installation with the one-line script for Debian-based systems
---------------------------------------------------------------
-
-.. note:: This script installs the latest development version, so be
-          careful not to use it on a production system.
-
-For a one-liner install of Zotonic and its dependencies you can use
-the following script::
-
-  wget -O - https://raw.github.com/zotonic/zotonic/master/zotonic_install | bash
-
-This will install all dependencies, clone the Zotonic `git`
-repository, create a ``zotonic`` user, add the necessary database
-permissions, and compile and start Zotonic.
-
-Now, point your browser to http://localhost:8000/ and make sure
-you see the `Powered by Zotonic` welcome screen. Then, head on to
-:ref:`installation-install-addsite`.
```

Comment:
Removed this because the script is outdated (still uses Wheezy) and we should work towards a real Zotonic package instead.